### ISSUE TITLE
Certificate x509 error with private registry

### ISSUE THREAD

**Issue by chalky9909** [aerokube/cm#120]
When launching cm (latest v1.3.2) as a docker container where the image is held in a private docker registry, the below error is being received:

```
> Using Docker
registry.ping url=https://docker-private.domain.com/v2/
Failed to initialize: [new configurator: Docker Registry is not available: Get https://docker-private.domain.com/v2/: x509: certificate signed by unknown authority]
```

The certificate is loaded on the OS (CentOS7) and docker images can be pulled down with no issues.

When launching using the cm binary using a syntax similar to the below on the same host, this works with no issues:

`DOCKER_API_VERSION=1.24 ./cm selenoid start --registry https://docker-private.domain.com:443
`
```
> Using Docker
registry.ping url=https://docker-private.domain.com:443/v2/
> Selenoid is already downloaded
> Selenoid is alredy configured
> Starting Selenoid...
2017-12-29T13:14:53.631345734Z container create 7e20033034b2c760397348df7133123fe27513bfeaef8708cce9e41480388a36 (image=docker-private.domain.com/aerokube/selenoid:1.4.1, name=selenoid)
2017-12-29T13:14:53.663953191Z network connect 4f694ab95b6d00d5aea01809f4b4cc7eed45ad127901950f0354fe2c308fb1c0 (container=7e20033034b2c760397348df7133123fe27513bfeaef8708cce9e41480388a36, name=bridge, type=bridge)
2017-12-29T13:14:53.780500161Z container start 7e20033034b2c760397348df7133123fe27513bfeaef8708cce9e41480388a36> Successfully started Selenoid
 (image=docker-private.domain.com/aerokube/selenoid:1.4.1, name=selenoid)
```

Our preference (if possible) is to use the docker image rather than the binary so is this possible with a private docker registry secured with SSL.

**Comment by vania-pooh** [comment]
I think this works as a binary because you have your internal SSL Root CA certificate installed in your operating system. When running as Docker container - we only install publicly available CA certificates. I see two possible solutions here:
1) Create your own `cm` container with the following Dockerfile:
```
FROM aerokube/cm:1.3.2

ADD YourCertFile.crt /usr/share/ca-certificates/mozilla/
```
2) Launch `cm` in container by adding a directory with SSL certs:
```
$ docker run -e SSL_CERT_DIR="/certs" -v /path/to/certs/dir:/certs [<rest of args>] aerokube/cm:1.3.2
```

**Comment by chalky9909** [comment]
@vania-pooh thank you for your quick response.

I have looked at the 2nd option and that has done the trick - thank you for pointing me in the right direction.

As we are using docker 1.12.6 (not docker-ce) I also had to add the below as a docker arg:
`-e DOCKER_API_VERSION=1.24
`
After adding the certs and api environment args to the docker run line this has now enabled the container to launch.

Thanks again.
